config/jobs/kubernetes/sig-k8s-infra/trusted/sig-contribex-tempelis.yaml

```
@@ -0,0 +1,30 @@
+postsubmits:
+  kubernetes/community:
+  - name: post-community-tempelis-apply
+    cluster: k8s-infra-prow-build-trusted
+    decorate: true
+    branches:
+    - ^master$
+    run_if_changed: '^communication/slack-config'
+    annotations:
+      testgrid-num-failures-to-alert: "1"
+      testgrid-alert-email: ktbry@google.com, k8s-infra-oncall@google.com
```

(non-blocking comment): drop or:
```suggestion
      testgrid-alert-email: sig-contribex-leads@kubernetes.io
```
?

---

feels like we should have a group email for the slack admins

---

moved to contribex leads with a TODO about slack admins list